plugins/mcreator-localization/help/default/entity/raid_spawn_percentage.md

```diff
@@ -0,0 +1,3 @@
+Here you can set how many times the entity will spawn in each raid wave.

```

Comment:
You should also explain this in the help tip:

![image](https://github.com/MCreator/MCreator/assets/16374228/d3dd32f4-65e7-40d1-ac13-0e22eef0b4f0)

Reply:
Done!